src/startUpdateDependents.ts

```diff
@@ -34,7 +34,7 @@ export default async function startUpdateDependents({ cwd }: { cwd: string }) {
     message: `Ticket to associate with pull requests? (e.g. ON-4323, OB-4323)`,
     required: true,
     validate: (input) => {
-      if (!/^(ON-|OB-)\d+$/.test(input)) {
+      if (!/^(ON-|OB-)\d+$/i.test(input)) {
```

Comment:
Can we please update the result to uppercase?

Reply:
oui oui we can do such a thing